roles/zabbix_agent/templates/zabbix_agentd.conf.j2

```diff
@@ -1,221 +1,63 @@
 {{ ansible_managed | comment }}
-# this is a config file for zabbix agent
-# to get more information about zabbix, visit http://www.zabbix.com
+# This is a configuration file for Zabbix Agent
+# To get more information about Zabbix, visit http://www.zabbix.com
 
-############ general parameters #################
+# This configuration file is "minimalized", which means all the original comments 
+# are removed. The full documentation for your Zabbix Agent can be found here:
+# https://www.zabbix.com/documentation/{{ zabbix_version }}/en/manual/appendix/config/zabbix_agentd{{ "_win" if zabbix_agent_os_family == "Windows" else "" }}
 
-### option: pidfile
-#       name of pid file.
-#
+{{ ansible_managed | comment }}
```

Comment:
Duplicate ansible_managed block? One present on line 1

Reply:
Yup that was a mistake. Fixed, thanks!